### PR TITLE
Vietnamese translation of src/docs/catalog/samples/basic-app-bar.md

### PR DESCRIPTION
**Pull request by khmai**

### CODE REVIEW

File: src/docs/catalog/samples/basic-app-bar.md [hunk 1]
```diff
@@ -23,21 +23,22 @@ A typical AppBar with a title, actions, and an overflow dropdown menu.
...
-Try this app out by creating a new project with `flutter create` and
-replacing the contents of `lib/main.dart` with the following code.
+Hãy dùng thử ứng dụng này bằng cách tạo một dự án mới với `flutter create` và
+thay thế nội dụng trong `lib/main.dart` bằng đoạn mã nguồn sau đây.
 
 ```dart
 // Copyright 2017 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
+// Việc sử dụng đoạn mã nguồn này bị chi phối bởi giấy phép BSD-style được nêu trong
```
Comment (truongsinh) [inline src/docs/catalog/samples/basic-app-bar.md]: do not translate copyright header

**Review by truongsinh — CHANGES_REQUESTED** [review]
please at least revert the translation on the copyright header. For the inline comment, let's wait to see whether we need to do the translation.

**Comment by duynht** [comment]
LGTM. I think we can merge @truongsinh

**Review by duynht — APPROVED** [review]

File: src/docs/catalog/samples/basic-app-bar.md (hunk 1 above)
Comment (duynht) [reply src/docs/catalog/samples/basic-app-bar.md]: It's been fixed

**Pull request merged**